### ISSUE TITLE
Build failure: vulkan-helper

### ISSUE THREAD

**Issue by Nosirus** [NixOS/nixpkgs#256456]
When wanting to update Heroic, he asks me for vulkan-helper.
but it seems cargoSha256 is bad.

```console
error: hash mismatch in fixed-output derivation '/nix/store/5m2zc2ncrgwvsc0h0ax9429rrqn0yimh-vulkan-helper-unstable-2023-09-16-vendor.tar.gz.drv':
         specified: sha256-s5QytuNhjZQhIDJtpeAW3J4op1t4nC+xD2i7Zf5mzfw=
            got:    sha256-fgB0vlbOhzGV1Sj180GCuTGZlVpAUlBUMAfsrG2FiuA=
```

**Comment by Artek52** [comment]
It has been fixed in #255708 but It must still arrive on nixos unstable

**Comment by lelgenio** [comment]
Fixed in nixpkgs-unstable

```
$ date --rfc-2822 --utc
Fri, 22 Sep 2023 00:16:54 +0000
$ nix build nixpkgs-unstable#vulkan-helper && echo it works!
it works!
```
